src/components/partner-logo/partner-logo.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import logoBig from "./assets/logo.png";
```

Comment:
Hey @victorbojica I just noticed that we have a new logo for the `In parteneriat cu`.  Similar to https://stirioficiale.ro/informatii

You can find it on slack in [this](https://codeforromania.slack.com/archives/GV1MGPGP4/p1584434760124800) post. Do you mind updating it, please?

Reply:
here's the new logo https://github.com/code4romania/covid-assets/blob/master/gov.png

Reply:
sure